src/aead/aes.rs

```diff
@@ -356,7 +355,7 @@ impl From<Counter> for Iv {
 
 impl Iv {
     pub(super) fn as_bytes_less_safe(&self) -> &[u8; 16] {
-        self.0.as_byte_array()
+        zerocopy::transmute_ref!(&self.0)
```

Comment:
Once we import in the benchmarks for `ring::{digest,hmac,pbkdf2}` then we can evaluate the performance of PR #1731. I expect that even if PR #1731 as written has performance issues, we will be able to tweak it so it doesn't. This will almost definitely eliminate this code completely.